kernel/os/include/os/os_mempool.h

```diff
@@ -188,6 +188,15 @@ os_error_t os_mempool_init(struct os_mempool *mp, uint16_t blocks,
 os_error_t os_mempool_ext_init(struct os_mempool_ext *mpe, uint16_t blocks,
                                uint32_t block_size, void *membuf, char *name);
 
+/**
+ * Clears a memory pool.
+ *
+ * @param mp            Pointer to a pointer to a mempool
```

Comment:
Too many levels of indirection in this comment :). How about: "Mempool to clear"?

Reply:
copy and paste :) will fix that